_posts/developers/apps/inrupt-tutorial/2020-11-06-00_getting-started.md

```diff
@@ -250,40 +266,24 @@ In the `my-demo-app` directory, create the files for the application.
       setStringNoLocale,
       saveSolidDatasetAt
     } from "@inrupt/solid-client";
-    
     import { Session } from "@inrupt/solid-client-authn-browser";
-    
     import { VCARD } from "@inrupt/vocab-common-rdf";
-    
-    // If your Pod is *not* on `solidcommunity.net`, change this to your Solid identity provider.
+
+    // If your Pod is *not* on `solidcommunity.net`, change this to your identity provider.
     const SOLID_IDENTITY_PROVIDER = "https://solidcommunity.net";
-    document.getElementById("solid_identity_provider").innerHTML = `[<a target="_blank" href="${SOLID_IDENTITY_PROVIDER}">${SOLID_IDENTITY_PROVIDER}</a>]`;
-  
-    const NOT_ENTERED_WEBID = "...not logged in yet - but enter any WebID to read from its profile...";
-    
+    document.getElementById(
+      "identity_provider"
+    ).innerHTML = `[<a target="_blank" href="${SOLID_IDENTITY_PROVIDER}">${SOLID_IDENTITY_PROVIDER}</a>]`;
+
+    const NOT_ENTERED_WEBID =
+      "...not logged in yet - but enter any WebID to read from its profile...";
+
     const session = new Session();
-    
+
     const buttonLogin = document.getElementById("btnLogin");
-    const buttonWrite = document.getElementById("btnWrite");
-    const buttonRead = document.getElementById("btnRead");
-  
-    // Click button if the user hits the 'Enter' key when entering name.
-    document.getElementById("input_name").addEventListener("keyup", event => {
-      if(event.key === "Enter") {
-        buttonWrite.click();
-        event.preventDefault();
-      }
-    });
-    
-    // Click button if the user hits the 'Enter' key when entering WebID.
-    document.getElementById("webID").addEventListener("keyup", event => {
-      if(event.key === "Enter") {
-        buttonRead.click();
-        event.preventDefault();
-      }
-    });
-    
-    
+    const writeForm = document.getElementById("writeForm");
+    const readForm = document.getElementById("readForm");
+
```

Comment:
Just noticed, in the login function below - do we need the 
!new URL(window.location.href).searchParams.get("code") still?

Reply:
Yeah good question - I think not. @pmcb55, do you know?

I'll give it a try in a sec without it.

Reply:
Seems to work fine without it, so I've removed it: 2ee53e0